stdlib/public/runtime/ImageInspectionCOFF.h

```diff
@@ -21,46 +21,12 @@
 
 #if !defined(__ELF__) && !defined(__MACH__)
 
-#include "../SwiftShims/Visibility.h"
 #include <cstdint>
-#include <cstddef>
 
 namespace swift {
-struct SectionInfo {
-  uint64_t size;
-  const char *data;
-};
-
 static constexpr const uintptr_t CurrentSectionMetadataVersion = 1;
```

Comment:
This can probably be shuffled along with the the rest of the definitions (and should be the same on ELF and COFF already).

Reply:
Ok! I figured we might want two different versions (one for ELF, one for COFF), that's why I left it in.

Reply:
@compnerd Since I'm removing this, I can delete both `ImageInspectionCOFF.h` and `ImageInspectionELF.h` right?

Reply:
Precisely.